Project.toml

```diff
@@ -1,23 +1,18 @@
-name = "ChainRules"
-uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "0.0.1"
+name = "AbstractChainRules"
+uuid = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
+version = "0.1.0"
 
 [deps]
 Cassette = "7057c7e9-c182-5462-911a-8362d720325c"
-LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-NaNMath = "77ba4419-2d1f-58cd-9bb1-8ffee604a2e3"
-SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
-Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+
 
 [compat]
 Cassette = "^0.2"
-FDM = "^0.6"
 julia = "^1.0"
 
 [extras]
-FDM = "e25cca7e-83ef-51fa-be6c-dfe2a3123128"
-Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 
 [targets]
-test = ["FDM", "Random", "Test"]
+test = ["Test", "LinearAlgebra"]
```

Comment:
I'm not sure what tests we have that depend on `LinearAlgebra`. Could we maybe remove this?

Reply:
We need it for Diagonal so we can test how goes on a random matrix type